### ISSUE TITLE
Wiki - some pages not cloning properly

### ISSUE THREAD

**Issue by Kerbas-ad-astra** [BobPalmer/MKS#1133]
I'm working on making some offline backups of KSP wikis that I often reference or expect to reference (since Internet access is sometimes a little flaky where I live), but several wiki pages have colons in their names, which causes Windows to choke on it while cloning. Can the wiki team (e.g. @notthebobo) please change that?

**Comment by notthebobo** [comment]
Googling showed that colons are not recommended in the wiki page names so I renamed the pages with colons, replacing them with parenthesis, which appears to work with downloads (on Windows at least).  I attempted to update the interpage links, so hopefully nothing is broken.

**Comment by Kerbas-ad-astra** [comment]
Yeah, that's where the "ZAPPED AS OBE" pages at the bottom of the list come from.  It might be good to scrub through them and ensure all the useful content has been pulled out of them, but from the few I checked through, it looks like their content is outdated or already mirrored on 'proper' pages.  Thanks!

**Comment by notthebobo** [comment]
I removed the colons from the ZAPPED pages.  I left a note for RD in the MKS Documentation thread asking if he minds the pages being removed.  If not, I'll go ahead and delete them.  If I don't hear from him in a few days, I'll try the MKS thread.
